src/App/Tree.js

```diff
@@ -56,7 +55,7 @@ function Tree(
     if (d.x < x0) x0 = d.x;
   });
 
-  // if (height === undefined) height = x1 - x0 + dx * 2;
+  if (height === undefined) height = x1 - x0 + dx * 2;
 
   if (typeof curve !== "function") throw new Error(`Unsupported curve`);
```

Comment:
따옴표로 통일하면 보기 좋을것 같습니다!
```suggestion
  if (typeof curve !== "function") throw new Error("Unsupported curve");
```